api/src/main/java/org/pmiops/workbench/institution/PublicInstitutionDetailsMapper.java

```diff
@@ -0,0 +1,10 @@
+package org.pmiops.workbench.institution;
+
+import org.mapstruct.Mapper;
+import org.pmiops.workbench.db.model.DbInstitution;
+import org.pmiops.workbench.model.PublicInstitutionDetails;
+
+@Mapper(componentModel = "spring")
+public interface PublicInstitutionDetailsMapper {
+  PublicInstitutionDetails dbToModel(DbInstitution dbObject);
```

Comment:
Don't even need to specify the mappings!
This is because it's a subset and the names are the same.